pygeoapi/api.py

```diff
@@ -744,7 +750,9 @@ def execute_process(self, headers, args, data, process):
             LOGGER.error(exception)
             return headers_, 400, json.dumps(exception)
 
-        if process not in self.config['processes'].keys():
+        processes = self.config.get('processes', {})
+
+        if not processes or process not in processes.keys():
```

Comment:
same here: `processes` will always have `dict` value so check could be `if process not in processes.keys():`..